ftplugin/NeogitCommitMessage.lua

```diff
@@ -0,0 +1,16 @@
+if vim.b.did_ftplugin then
+  return
+end
+
+vim.cmd.source("$VIMRUNTIME/ftplugin/gitcommit.vim")
+vim.cmd([[exec 'norm! gg']])
```

Comment:
In which situations it the cursor not at the top?

Reply:
Probably none for most people, but if you have a "remember cursor location in buffer" autocmd/plugin, this would attempt to defeat that.

Reply:
Ditched this. Probably not needed after all.